rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -17,6 +17,14 @@ function _M.get_general_data()
   return configuration_data:get("general")
 end
 
+function _M.get_timestamp_data()
```

Comment:
Similar to https://github.com/kubernetes/ingress-nginx/pull/5418/files#r412949876, `raw_backends_last_sycned_at`.